includes/discovery/sensors/temperatures/qnap.inc.php

```diff
@@ -23,9 +23,14 @@
             // Get the disk temperature full oid
             $disk_oid = $disk_temperature_oid.$disk_number;
             // Get the temperature for the disk
-            $disk_temperature = $entry['HdTemperature'];
+            $disk_temperature = $entry['hdTemperatureEX'];
```

Comment:
Are we sure that the old `HdTemperature` isn't version dependant and this won't break it for other users?